commit d2d12c7a988a9a04aec23b5e4af549db61b0a005
Author: H.J. Lu <hjl.tools@gmail.com>
Date:   Tue Feb 2 09:31:56 2021 -0800

    tst-rtld-list-tunables.sh: Unset glibc tunables
    
    Unset glibc tunables and their aliases for --list-tunables test.

diff --git a/elf/tst-rtld-list-tunables.sh b/elf/tst-rtld-list-tunables.sh
index e7bbdde94952b872..78f4ed2ebbd3db2c 100755
--- a/elf/tst-rtld-list-tunables.sh
+++ b/elf/tst-rtld-list-tunables.sh
@@ -26,6 +26,17 @@ run_program_env=$3
 LC_ALL=C
 export LC_ALL
 
+# Unset tunables and their aliases.
+GLIBC_TUNABLES=
+MALLOC_ARENA_MAX=
+MALLOC_ARENA_TEST=
+MALLOC_CHECK_=
+MALLOC_MMAP_MAX_=
+MALLOC_MMAP_THRESHOLD_=
+MALLOC_PERTURB_=
+MALLOC_TOP_PAD_=
+MALLOC_TRIM_THRESHOLD_=
+
 ${test_wrapper_env} \
 ${run_program_env} \
 $rtld --list-tunables \
